### PR TITLE
[CDS- 485] Product documentation v2

### PR DESCRIPTION
**Pull request by michael-le-19**
# Merge request description template

## Description

Create new page: Product in docs v2

**1) Changes**

Create new page: Product in docs v2

MicrosoftTeams-image (1)

https://comfortdelgrotaxi.atlassian.net/browse/CDS-485

**2) Impact**

## Check list

- [x] 1. Did you start and verify your changes?
- [x] 2. Did you run build to make sure that your changes can be built successfully?
- [x] 3. No !important flag, inline style in css
- [x] 4. No bolerplate codes (1)
- [x] 5. No duplicate code that exist in common functions?
- [x] 6. All of defined variables should have default value, check null and undefined?
- [x] 7. Use meaningful name for variables, no suffix 1,2,... Describes reference information for easier to understand what's inside. (2)
- [x] 9. Unsubscribed all of subscribers and even listeners when you don't need them.

```
(1)
Boilerplate codes is duplicated multiple times a bunch of the same code. This should be a common function to reuse through your code or you can use generic class / methods or design pattern to avoid the Boilerplate codes.
```

```
(2)
Follow theo coding convention
NO acronym variable name:
WRONG: el, elm
RIGHT: element
Your code can be read as a sentence
```

### CODE REVIEW

File: apps/docs-v2/components/products/ProductServices.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  FMSLogo,
+  FNBLogo,
+  TEMLogo,
+  ZigStoreLogo,
+} from './ProductLogos'
+
+const ProductService = () => {
+  const lorem = 'Lorem ipsum dola sign lewded saedh kdkroe krtyer'
+
+  const products = useMemo<ProductCard[]>(() => {
```
Comment (hoapn-fsoft) [inline apps/docs-v2/components/products/ProductServices.tsx:17]: No `useMemo` needed and let's move it outside the component

File: apps/docs-v2/pages/products/index.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+import ProductsBackground from 'components/products/ProductsBackground'
+import Head from 'next/head'
+
+const Products = () => {
+  return (
+    <>
+      <Head>
+        <title>Products - React Compass</title>
+      </Head>
+      <Column
```
Comment (hoapn-fsoft) [inline apps/docs-v2/pages/products/index.tsx:13]: why don't you use grid?

File: apps/docs-v2/components/products/ProductServices.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    <Grid
+      css={{
+        maxWidth: '1200px',
+        width: '100%',
+        backgroundColor: '$gray10',
+        padding: '0 0 24px 24px',
+        marginTop: '24px',
+        borderRadius: '24px',
+        boxShadow:
+          '0px 3.200000047683716px 7.199999809265137px 0px rgba(0, 0, 0, 0.13), 0px 0.6000000238418579px 1.7999999523162842px 0px rgba(0, 0, 0, 0.10)',
```
[the remark this answers is not in the document]
Comment (michael-le-19) [reply apps/docs-v2/components/products/ProductServices.tsx:80]: it's the number in figma design

File: apps/docs-v2/pages/products/index.tsx (hunk 2 above)
Comment (michael-le-19) [reply apps/docs-v2/pages/products/index.tsx:13]: i use grid in repetitive layout

File: apps/docs-v2/components/products/ProductServices.tsx (hunk 1 above)
Comment (michael-le-19) [reply apps/docs-v2/components/products/ProductServices.tsx:17]: will do

File: apps/docs-v2/components/products/ProductServices.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  CarbyPortalLogo,
+  CMSLogo,
+  ComfortConnectLogo,
+  FMSLogo,
+  FNBLogo,
+  TEMLogo,
+  ZigStoreLogo,
+} from './ProductLogos'
+
+const ProductService = () => {
```
[the remark this answers is not in the document]
Comment (michael-le-19) [reply apps/docs-v2/components/products/ProductServices.tsx:14]: will do

**Review by hoapn-fsoft — APPROVED** [review]

**Pull request merged**